### DISCUSSION TITLE
Mailchimp server that send out emails to maintainers when a package is outdated

### DISCUSSION THREAD

**Discussion by wizard-28** [pacstall/website#15] [Ideas]
- how do users opt-in for this?
- how do we prevent users from using other people's email?

Possible solutions:
 - use GitHub for authentications and only notify verified emails
- same as above but without GitHub. use our own auth

Issue created by @Henryws

**Comment by wizard-28** [comment]
Why not just tell the maintainers to turn on GitHub/GitLab release notifications for the package they're maintaining?

**Comment by Elsie19** [comment]
This was done with GitHub issues and pacbot.
